packages/cli/src/WorkflowRunnerProcess.ts

```diff
@@ -174,8 +174,8 @@ export class WorkflowRunnerProcess {
 				},
 			],
 			nodeExecuteAfter: [
-				async (nodeName: string, data: ITaskData, executionData: IRunExecutionData): Promise<void> => {
-					this.sendHookToParentProcess('nodeExecuteAfter', [nodeName, data, executionData]);
+				async (nodeName: string, data: ITaskData): Promise<void> => {
```

Comment:
One important note that I would like to add here is that by removing the `executionData` from the message that is sent to parent, we're not following the signature for `nodeExecuteAfter` functions.

On the `Intefaces.ts` file in the workflow project we declare it like this:

```
nodeExecuteAfter?: Array<((nodeName: string, data: ITaskData, executionData: IRunExecutionData) => Promise<void>)>;
```

But considering the performance issues, it's better to remove it now as it is not currently being used by the parent process.